ulwgl_run.py

```diff
@@ -2,6 +2,7 @@
 
 import os
 import argparse
+import traceback
 from argparse import ArgumentParser, Namespace
```

Comment:
I think we should try to import what we need here for now.

> from traceback import print_tb

Or probably better, just [print_exception](https://docs.python.org/3/library/traceback.html#traceback.print_exception)

> from traceback import print_exception

Reply:
agreed